src/components/Calculator.js

```diff
@@ -20,39 +21,125 @@ Output.defaultProps = {
       <div className="calcContainer">
-        <Output result={0} />
+        <Output result={disp} />
         <div className="keysGrid">
-          <Button className="calcBtn" value="AC" />
-          <Button className="calcBtn" value="+/-" />
-          <Button className="calcBtn" value="%" />
-          <Button className="calcBtn" value="+" />
-          <Button className="calcBtn" value="7" />
-          <Button className="calcBtn" value="8" />
-          <Button className="calcBtn" value="9" />
-          <Button className="calcBtn" value="x" />
-          <Button className="calcBtn" value="4" />
-          <Button className="calcBtn" value="5" />
-          <Button className="calcBtn" value="6" />
-          <Button className="calcBtn" value="-" />
-          <Button className="calcBtn" value="1" />
-          <Button className="calcBtn" value="2" />
-          <Button className="calcBtn" value="3" />
-          <Button className="calcBtn" value="+" />
-          <Button className="calcBtn" value="0" />
-          <Button className="calcBtn" value="." />
-          <Button value="=" />
+          <Button
+            value="AC"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="+/-"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="%"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="÷"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="7"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="8"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="9"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="x"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="4"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="5"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="6"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="-"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="1"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="2"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="3"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="+"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="0"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="."
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="="
+            handleClick={this.handleClick}
```

Comment:
- [ ] [Optional] It would be nice to use JavaScript `map` method to render the buttons. Using the map method will ensure that you don't repeat code and that you follow the DRY principle. You can learn more about using the map method in react by reading this [article](https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/). 👍